AntennaTracker/GCS_MAVLink_Tracker.cpp

```diff
@@ -63,6 +63,28 @@ MAV_STATE GCS_MAVLINK_Tracker::vehicle_system_status() const
     return MAV_STATE_ACTIVE;
 }
 
+void GCS_MAVLINK_Tracker::send_attitude_target()
+{
+    const Quaternion quat = tracker.mode_guided.get_attitude_target_quat();
+    const bool use_yaw_rate = tracker.mode_guided.get_attitude_target_use_yaw_rate();
+    const float yaw_rate_rads = tracker.mode_guided.get_attitude_target_yaw_rate_rads();
+    const float quat_out[4] {quat.q1, quat.q2, quat.q3, quat.q4};
+    uint16_t typemask = (1<<0) + (1<<1) + (1<<6);
+    if (!use_yaw_rate) {
+        typemask |= 1<<2;
+    }
+
+    mavlink_msg_attitude_target_send(
```

Comment:
This should only be sent in guided mode if that is the only mode where the targets are available.

Reply:
@IamPete1 thanks. I added ATTITUDE_TARGET support in the modes: scan, auto, stop, guided. And blocked sending in other modes